src/tqec/templates/display.py

```diff
@@ -29,6 +30,88 @@ def display_template(
         print()
 
 
+def display_templates_ascii(
+    templates: ComposedTemplate,
+    plaquette_indices: ty.Sequence[int] | None = None,
+    h_space: int = 5,
+    v_space: int = 2,
+    corner_mark: str = "■",
```

Comment:
```suggestion
    corner_mark: str = "+",
```

Having a non-unicode default separator seems like the best option as:
- it is in the direction of maximum compatibility,
- it still allow anyone to use unicode,
- it might avoid users thinking that unicode characters are perfectly handled. There might be very_few edge cases with the` len(corner_mark) == 1` check for strange unicode characters. These edge cases will likely never be triggered, but let's be on the safe side.